mscore/editharmony.cpp

```diff
@@ -99,6 +99,10 @@ void ScoreView::harmonyTab(bool back)
 void ScoreView::harmonyBeatsTab(bool noterest, bool back)
       {
       Harmony* harmony = toHarmony(editData.element);
+      if (!harmony->parent() || !harmony->parent()->isSegment()) {
+            qDebug("no segment parent");
```

Comment:
I guess the effect of this is, you can't use the navigation shortcuts (space, tab, etc) to move to the next position to enter chords if the current one is attached to a fret diagram?  That's OK for now if so, might be better eventually to change the behavior of space (or tab, or use a new shortcut) to actually find the next fret diagram.

Reply:
Yes, that'd be good. But this just stops it crashing for now.